store/src/java/com/zimbra/cs/service/account/GetHAB.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2018 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+package com.zimbra.cs.service.account;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.zimbra.common.account.Key;
+import com.zimbra.common.account.Key.DomainBy;
+import com.zimbra.common.service.ServiceException;
+import com.zimbra.common.soap.AccountConstants;
+import com.zimbra.common.soap.Element;
+import com.zimbra.common.util.ZimbraLog;
+import com.zimbra.cs.account.Account;
+import com.zimbra.cs.account.Provisioning;
+import com.zimbra.cs.account.ldap.entry.LdapDistributionList;
+import com.zimbra.soap.ZimbraSoapContext;
+import com.zimbra.soap.account.message.HABGroup;
+import com.zimbra.soap.type.ZmBoolean;
+
+/**
+ * @author zimbra
+ *
+ */
+public class GetHAB extends AccountDocumentHandler {
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see
+     * com.zimbra.soap.DocumentHandler#handle(com.zimbra.common.soap.Element,
+     * java.util.Map)
+     * 
+     */
+    @Override
+    public Element handle(Element request, Map<String, Object> context) throws ServiceException {
+        ZimbraSoapContext zsc = getZimbraSoapContext(context);
+        Provisioning prov = Provisioning.getInstance();
+        String rootGrpId = request.getAttribute(AccountConstants.A_HAB_ROOT_GROUP_ID);
+        Account acct = getAuthenticatedAccount(zsc);
+
+        ZimbraLog.account.info("Request for HAB group:%s by: ", rootGrpId, acct);
+
+        Map<String, HABGroup> groups = new HashMap<String, HABGroup>();
+        List<HABGroup> childGrpList = new ArrayList<HABGroup>();
+        LdapDistributionList group = (LdapDistributionList) prov.getGroupBasic(Key.DistributionListBy.id, rootGrpId);
```

Comment:
override the getGroupBasic with additional request parameter returnBasicAttrs, and call this method with value false.